apps/automation/automation-v2/src/tasks/spice-auction-burn-and-notify.ts

```diff
@@ -60,6 +61,24 @@ export async function redeemTempleGold(ctx: TaskContext, params: Params): Promis
     client: pclient
   });
 
+  // Overlord EOA approves spice contract to spend max TGLD if current approval is less than amount
```

Comment:
Why does it grant the spice contract approval? 

That's 
config.contracts.TEMPLE_GOLD.SPICE_ENA_TGLD correct? How would that contract pull it back?